frontend/client/components/ErrorScreen/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import * as Sentry from '@sentry/browser';
+import { Button } from 'antd';
+import Exception from 'ant-design-pro/lib/Exception';
+import './index.less';
+
+interface Props {
+  error: Error;
+}
+
+export default class ErrorScreen extends React.PureComponent<Props> {
+  componentDidMount() {
+    const { error } = this.props;
+    Sentry.captureException(error);
+    console.error('Error screen showing due to the following error:', error);
+  }
+
+  render() {
+    const { error } = this.props;
+    return (
+      <div className="ErrorScreen">
+        <Exception
+          type="404"
+          title="Whoa nelly."
+          desc={
+            <div className="ErrorScreen-desc">
+              <p>Something went wrong, and we've logged the following error:</p>
+              <code className="ErrorScreen-desc-error">{error.message}</code>
+              <p>
+                Our developers will get right on fixing it. You can either return home and
+                try again, or open an issue on Github to provide us some more details.
+              </p>
+            </div>
+          }
+          actions={
+            <div className="ErrorScreen-buttons">
+              <a href="/">
```

Comment:
`<Link to="/">...`?

Reply:
This is intentional, to completely reload state (assuming something ephemeral went wrong, like an uncaught server request.)